src/status_im2/contexts/quo_preview/selectors/selectors.cljs

```diff
@@ -1,91 +1,73 @@
-            [reagent.core :as reagent]
-            [status-im.ui.components.react :as react]))
+  (:require
+    [quo2.components.markdown.text :as text]
+    [quo2.components.selectors.selectors :as quo2]
+    [quo2.foundations.colors :as colors]
+    [react-native.core :as rn]
+    [reagent.core :as reagent]
+    [status-im2.contexts.quo-preview.preview :as preview]))
 
 (def descriptor
-  [{:label "Disabled?"
+  [{:label "Disabled"
     :key   :disabled?
-    :type  :boolean}])
+    :type  :boolean}
+   {:label "Blur"
+    :key   :blur?
+    :type  :boolean}
+   {:label   "Custom color"
+    :key     :custom-color
+    :type    :select
+    :options (map (fn [[color _]]
+                    {:key color :value (name color)})
+                  colors/customization)}])
+
+(defn selector-preview
+  [text component {:keys [disabled? blur? custom-color]}]
+  [rn/view
+   {:style {:margin      6
+            :align-items :center}}
+   [text/text {:size :paragraph-1} text]
+   [component
+    {:container-style {:margin 4}
+     :disabled?       disabled?
+     :blur?           blur?
+     :custom-color    custom-color}]])
 
 (defn cool-preview
   []
-  (let [state (reagent/atom {:disabled false})]
+  (let [state (reagent/atom {:disabled?    false
+                             :checked?     false
+                             :blur?        false
+                             :custom-color :blue})]
     (fn []
       [rn/view
-       {:margin-bottom 50
-        :padding       16}
        [preview/customizer state descriptor]
+       [rn/view {:style {:margin-vertical 24}}
+        [preview/blur-view
+         {:style                 {:width       "100%"
+                                  :align-items :center
+                                  :top         (if (:blur? @state) 32 16)
+                                  :position    (if (:blur? @state)
+                                                 :absolute
+                                                 :relative)}
+          :height                300
+          :show-blur-background? (:blur? @state)}
 
-       [rn/view
-        {:padding-vertical 60
-         :align-items      :center}
-        [text/text {:size :heading-2} "Toggle"]
-        [quo2/toggle
-         {:container-style {:margin-top 0}
-          :disabled?       (:disabled? @state)}]
-        [text/text {:size :heading-2} "Radio"]
-        [quo2/radio
-         {:container-style {:margin-top 0}
-          :disabled?       (:disabled? @state)}]
-        [text/text {:size :heading-2} "Checkbox"]
-        [quo2/checkbox
-         {:container-style {:margin-top 0}
-          :disabled?       (:disabled? @state)}]
-        [text/text {:size :heading-2} "Checkbox Prefill"]
-        [quo2/checkbox-prefill
-         {:container-style {:margin-top 0}
-          :disabled?       (:disabled? @state)}]]
-
-       [rn/view
-        {:padding-vertical 60
-         :align-items      :center}
-        [react/blur-view
```

Comment:
Due to the way this component was used, the screen had some kind of overlay even in the part of the view that wasn't using blur:
![image](https://user-images.githubusercontent.com/90291778/228973851-07176161-9a38-4c2e-b99e-68e80d431c46.png)

I removed it and now we're using our common preview blur component (when blur is applied:
![image](https://user-images.githubusercontent.com/90291778/228974032-2d8ef49d-623b-4c84-9ddd-3f3fd02cca56.png)